examples/vertex_ai_orchestration/README.md

```diff
@@ -0,0 +1,148 @@
+
+```bash
+# Create a zenml root
+zenml init
+ 
+# In order to create the GCP stack components, we'll need to install one 
+# additional ZenML integration:
+zenml integration install gcp
+
+# The CONTAINER_REGISTRY_URI will have a format like this: eu.gcr.io/xxx/xxx
+zenml container-registry register gcp_registry --flavor=gcp --uri=<CONTAINER_REGISTRY_URI>
+
+# The DB_HOST_IP is the public IP Address of your Database: xx.xx.xxx.xxx
+#  The DB_PORT is 3306 by default - set this in case this default does not apply
+#  The DB_NAME is the name of the database that you have created in GCP as the
+#  metadata store. The `mysql_secret` will be created once the secret manager
+#  is created and the stack is active.
+zenml metadata-store register gcp_metadata_store --flavor=mysql --host=`<DB_HOST_IP>` --port=`<DB_PORT>` --database=`<DB_NAME>` --secret=mysql_secret
+  
+# The PATH_TO_YOUR_GCP_BUCKET is the path to your GCP bucket: gs://xxx
+zenml artifact-store register gcp_artifact_store --flavor=gcp --path=`<PATH_TO_YOUR_GCP_BUCKET>`
+
+# The orchestrator needs the PROJECT_ID and the GCP_LOCATION in which to
+#  run the vertex ai pipeline. Additionally you might need to set the 
+#  WORKLOAD_SERVICE_ACCOUNT to the service account you created with secret
+#  manager access, it will be in the format: xxx@xxx.iam.gserviceaccount.com
+zenml orchestrator register vertex_orch --flavor=vertex --project=`<PROJECT_ID>` --location=`<GCP_LOCATION>`
+
+# For the secret manager, all we'll need it the gcp PROJECT_ID
+zenml secrets-manager register gcp_vertex_secrets_manager --flavor=gcp --project_id=`<PROJECT_ID>`
+
+# Now we're ready to assemble our stack
+zenml stack register gcp_vertex_stack -m gcp_metadata_store -a gcp_artifact_store -o vertex_orch -c gcp_registry -x gcp_vertex_secrets_manager --set
+
+# With the stack up and running, we can now supply the credentials for the 
+#  mysql metadata store. The SSL certificates have to be generated and downloaded
+#  from within the CloudSQL UI
+zenml secret register mysql_secret --schema=mysql --user=`<DB_USER>` --password=`<PWD>` \
+  --ssl_ca=@`</PATH/TO/DOWNLOADED/SERVER-CERT>` \
+  --ssl_cert=@`</PATH/TO/DOWNLOADED/CLIENT-CERT>` \
+  --ssl_key=@`</PATH/TO/DOWNLOADED/CLIENT-KEY>`
+```
+
+Your stack should look something like this when you're done:
+
+![Vertex Stack](assets/vertex_stack.png)
+
+### ▶️ Run the pipeline
+
+Once your stack is fully set up, you should be good to go. 
+
+```bash
+python run.py
+```
+
+That's it! If everything went as planned this pipeline should now be running in
+the cloud! You should be able to access the Vertex Ai Pipelines UI with the link
+returned to the run logs. It will look something like this:
+
+![Vertex AI UI](assets/vertex_ai_ui.png)
+
+### 🧽 Clean up
+
+Once you're done experimenting, you can stop the port forwarding and delete the
+example files by calling:
+
+```bash
+zenml stack down --force
+rm -rf zenml_examples
```

Comment:
I'll just add the comment here with the cleanup, but this cleanup only makes sense if we include the usual
```shell
zenml example pull vertex_ai_orchestration
cd zenml_examples/vertex_ai_orchestration
```
right?

Reply:
Correcto